lisa-utils/src/main/scala/lisa/utils/OutputManager.scala

```diff
@@ -1,9 +1,43 @@
 package lisa.utils
+import lisa.utils.Helpers.show
+
+import java.io.PrintWriter
+import java.io.StringWriter
 
 abstract class OutputManager {
+  given OutputManager = this
+
+  def output(s: String): Unit = stringWriter.write(s + "\n")
+  def output(s: String, color: String): Unit = stringWriter.write(color + s + "\n" + Console.BLACK)
```

Comment:
`BLACK` may not always be the default color for console printing, so this will lead to inconsistencies in printing theorems after the first.

Reply:
To be fixed next PR!